src/llmq/quorums_instantsend.cpp

```diff
@@ -820,6 +820,39 @@ void CInstantSendManager::RetryLockMempoolTxs(const uint256& lockedParentTx)
             }
         }
     }
+
+    uint256 lastChainLockBlock;
+    const CBlockIndex* pindexLastChainLockBlock = nullptr;
+    const CBlockIndex* pindexWalk = nullptr;
+    {
+        LOCK(cs);
+        lastChainLockBlock = db.GetLastChainLockBlock();
```

Comment:
can bail out early here
```
if (lastChainLockBlock.IsNull()) {
    return;
}
```

Reply:
Not really, as we might have entries in `txs` from the mempool loop at the top.